develop/TOOLS/DIS/Tutorials/DisMibBrowser.md

```diff
@@ -0,0 +1,83 @@
+---
+uid: DisTutorials_MibBrowser
+---
+
+# Tutorial: Exploring the DIS MIB Browser
+
+In this tutorial, you familiarize yourself with the DIS MIB Browser's functionalities by loading a MIB file, generating parameters, and comparing them to a sample protocol.
```

Comment:
I have replaced all instances of the term "connector" with "protocol". This is the term used in the DIS MIB Browser UI as well as the chosen name for the sample file uploaded to GitHub.

![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/a7d23e59-c0c9-4c94-be3d-ab0fc35beea0)


![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/a6c8bf74-5fc3-4430-8508-81d34dd3316a)

To avoid confusion, we should use one term only.

Reply:
I've replaced it back to connector as that is the term that should be used. With the gifs, it clarifies that protocol & connector are the same. 
I'll check with the team if we can modify DIS to also use more the term 'connector'.

Reply:
That would be great! If "protocol" were to become "connector" in the DIS UI, we will have to remember to update any related screenshots (e.g. [DIS_ToolWindow_MibBrowser_Compare.png](https://github.com/SkylineCommunications/dataminer-docs/pull/1894/files#diff-28878fd5dd1e5ba190bf0b3050cca96c542e891c4004b48792b13be736fa230d)).